README.md

```diff
@@ -2,6 +2,90 @@
 
 # Parity UI
 
+A set of [Parity](https://github.com/ethcore/parity) built-in DApps and UI components.
+
+[What's Parity?](https://github.com/ethcore/parity)
+
+## List of projects
+
+* [DApps React Components](./components) - Re-usable set of React Components
+* [Home Page & TopBar](./home) - Parity Home Page & TopBar
+* [Status Page](./status) - Status Page for your Parity node
+* [Ethereum Wallet](./wallet) - Ethereum Wallet bundled for Parity
+* [Trusted Signer](./signer) - Trusted Signer Dapp
+* [Trusted Signer Chrome Extension](./extension) - Parity Integration Chrome Extension
+
+### How to add new Builtin-DApp?
```

Comment:
typo / grammar
> How to add *a* new Building DApp?
> How to add new Building DApp*s*?

I also prefer Dapp over DApp, tho that's purely personal / subjective